samples/snippets/src/main/java/product/RemoveFulfillmentPlaces.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * [START retail_remove_fulfillment_places]
+ */
+
+package product;
+
+import com.google.cloud.retail.v2.ProductServiceClient;
+import com.google.cloud.retail.v2.RemoveFulfillmentPlacesRequest;
+import com.google.protobuf.Timestamp;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+
+import static setup.SetupCleanup.createProduct;
+import static setup.SetupCleanup.deleteProduct;
+import static setup.SetupCleanup.getProduct;
+
+public final class RemoveFulfillmentPlaces {
+
+  /**
+   * This variable describes project number getting from environment variable.
+   */
+  private static final String PROJECT_NUMBER = System.getenv("PROJECT_NUMBER");
+
+  /**
+   * This variable describes defined product id for field setting.
+   */
+  private static final String PRODUCT_ID = "remove_fulfillment_test_product_id";
```

Comment:
Use an UUID or use tryToDeleteProductIfExists before the test?

Reply:
it was decided to use the same approach for SetInventory, AddFulfillmentInfo and RemoveFulfillmentInfo code samples, which is using tryToDeleteProductIfExists